config.toml

```diff
@@ -63,6 +63,10 @@ show_tags = true
 # ["orange", "blue", "red", "green", "pink"]
 theme_color = "orange"
 
+# Whether or not to inline CSS in the generated HTML. Can provide
+# a performance improvement for some sites.
```

Comment:
```suggestion
# a performance improvement for some sites.
# This is useful for sites where often only one page is visited by a user.
# In such a case, separately loading/caching the CSS provides no extra 
# benefit, and slows down content rendering while waiting the CSS network 
# request to start and finish.
```

Reply:
i just took your PR summary and inserted it directly into your config change. does that work well enough to better explain the optional config feature?

Reply:
Yup, that works for me.